etc/licscripts/find_invlic.pl

```diff
@@ -71,9 +83,12 @@ ()
       return;
     }
   }
-
-  printf("** File %s did NOT match any license\n", $File::Find::dir . "/" . $entry);
-  $ok = 0;
+  if ( ! $use_known_files or File::Spec->rel2abs($entry) ~~ @known_files) { 
```

Comment:
This results in a warning about Smartmatch and after a quick search, this warning seems to have a good reason, as Smartmatch may be removed in the future. Instead, you can use 
```suggestion
  if ( ! $use_known_files or none { $_ =~ File::Spec->rel2abs($entry) } @known_files) {
```